packages/Organisms/Publication/PublicationDownloadPrint.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <ul class="rpl-publication-download-print">
+    <li v-for="(link, index) in links" :key="index">
+      <rpl-document-link class="rpl-publication-download-print__print-item" v-bind="link" />
+    </li>
+    <li v-if="showPrint" class="rpl-publication-download-print__print-list-item">
+      <button class="rpl-publication-download-print__print-item rpl-publication-download-print__print-button" @click="printClick()">
+        <rpl-icon class="rpl-publication-download-print__print-button-icon" symbol="print" color="primary" size="l" />
+        <div class="rpl-publication-download-print__print-button-info">
+          <span class="rpl-publication-download-print__print-button-title">{{ printText }}</span>
+        </div>
+      </button>
+    </li>
+  </ul>
+</template>
+
+<script>
+import RplIcon from '@dpc-sdp/ripple-icon'
+import DocumentLink from '@dpc-sdp/ripple-document-link'
+
+export default {
+  name: 'RplPublicationDownloadPrint',
+  components: {
+    RplIcon,
+    DocumentLink
+  },
+  props: {
+    links: Array,
+    showPrint: { type: Boolean, default: true },
+    printText: { type: String, default: 'Print this page' }
+  },
+  methods: {
+    printClick () {
+      window.print()
```

Comment:
As discussed, can we emit an event on print? Maybe called publicationPrint? That way we can use it for analytics tracking (I know people can invoke the print window separately too).

Reply:
Added an event.